app/controllers/media_objects_controller.rb

```diff
@@ -54,6 +55,26 @@ def confirm_remove
     raise CanCan::AccessDenied unless Array(params[:id]).any? { |id| current_ability.can? :destroy, MediaObject.find(id) }
   end
 
+  def intercom_collections
```

Comment:
Is this sufficiently authZ'd?  It is hard for me to tell from the before_actions.  A test could be added for this.
At second glance I'm guessing that a `can? :read, Admin:Collections` or something like that might need to be added.  Or maybe it doesn't matter because the user will only be getting back the collections they belong to in the other avalon?  
Do we have to worry about usernames not matching up between the avalons and privilege escalation?  Seems pretty unlikely but possible.

Reply:
The main driver for the auth choices here is the IU business rule that anyone who can view an object in MDPI can push it. I have thought about adding an ability for the push action that would encode this simple rule but easily could be overridden. On the actual ingest, it uses the ingest api which depends only on token auth, so the pushing user is irrelevant to the target.

Reply:
This method is probably safe because it is just returning information about the current user's account in the remote avalon.  The documentation for setting up intercom should include a note about the assumption that usernames are the same between the two avalons.